ca_on_toronto/people.py

```diff
@@ -46,9 +150,19 @@ def scrape_councilor(self, page, h1, url):
         if address:
             p.add_contact('address', address, 'legislature')
 
-        return p
+        yield p
 
     def scrape_mayor(self, url):
+        # TODO: Fix mayor scraper
```

Comment:
This is another issue to fix, but it's just a boring task, so I'm procrastinating and faking a fix for now :)